src/electionguard/serialize.hpp

```diff
@@ -539,7 +539,7 @@ namespace electionguard
                     serialized.push_back({"contact_information", contactInformation});
                 }
 
-                serialized.push_back({"spec_version", "1.0"});
+                serialized.push_back({"spec_version", "1.90"});
```

Comment:
I'm thinking this is causing the unit tests to fail.

Reply:
rolled back for now. We should split this into a separate PR, assuming tests pass